### PR TITLE
JSON Modification

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#82 
#### What does this implement/fix? Explain your changes.
- JSON format value changed from `string` to `number`
- YAML format value changed from `string` to `number`
- Pickle format value changed from `string` to `number`

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri 
Please review this PR 💯

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `requirements-splitter.py`
 ### Changed
 - Test system modified
+- JSON format value changed from `string` to `number`
+- YAML format value changed from `string` to `number`
+- Pickle format value changed from `string` to `number`
```
Comment (sadrasabouri) [inline CHANGELOG.md:14]: Isn't it better if we use `int` instead of `number`?

**Review by sadrasabouri — COMMENTED** [review]
I left a tiny suggestion for you.

File: CHANGELOG.md (hunk 1 above)
Comment (sepandhaghighi) [reply CHANGELOG.md:14]: the weight value is not always an integer.

**Pull request merged**